src/LodeRunner.Core/Models/LoadClient.cs

```diff
@@ -80,5 +80,20 @@ public static LoadClient GetNew(ILRConfig config, DateTime lastUpdated)
             config.LoadClientId = loadClient.Id;
             return loadClient;
         }
+
+        /// <summary>
+        /// Sets mock LoadClient data.
+        /// </summary>
+        /// <param name="name">LoadClient name.</param>
+        public void SetMockData(string name)
```

Comment:
Remove mock data generation out of the domain model.

Reply:
Addressed in commit by adding content to files in LodeRunner.API.Test/IntegrationTests/Extensions